core/client/fs/src/main/java/alluxio/client/block/AlluxioBlockStore.java

```diff
@@ -104,17 +103,25 @@ public BlockInfo getInfo(long blockId) throws IOException {
   }
 
   /**
-   * @return the info of all active block workers
+   * @return the info of all available block workers
```

Comment:
`Available` doesn't seem like an accurate description, `Eligible`?

Reply:
changed to Eligible